categoriesChoos.js

```diff
@@ -0,0 +1,69 @@
+var restaurantsToShow = inicializeRestaurantToShow();
+var idChoosenCategory = '';
+
+$(document).ready(function() {
+  idChoosenCategory = splittedLocationHash();
+  restaurantsToShow = inicializeRestaurantToShow();
+  showCategory();
+  moment.locale('pl');
+  $('#singInButton').on('click', function() {
+    $('#singInModal').modal('show');
+    localStorage.setItem('auth_token', 'aa');
+    localStorage.setItem('email', 'aaaa@ccc');
+    location.reload();
+  });
+
+  $('#singUpButton').on('click', function() {
+    $('#singUpModal').modal('show');
+  });
+
+  $('#linkMap').on('click', function() {
+    initMap();
+  });
+  if (localStorage.auth_token != undefined && localStorage.auth_token != '') {
+    $('.singInSingUp').html(
+      '<p>Jestes zalogowany jako ' +
+        localStorage.email +
+        '   <span class="singOut">Wyloguj sie</span></p>'
+    );
+    $('.singOut').on('click', function() {
+      localStorage.removeItem('auth_token');
+      localStorage.removeItem('email');
+      location.reload();
+    });
+  } else {
+    $('.addRateButton').prop('disabled', 'disabled');
```

Comment:
please let's move all jquery selectors to jquery variables.
jquery variables starts with a dollar sign (`$`) to notify that they represent a DOM element.
we use them, because everytime you call the whole selector it searches throughout the whole DOM for it which is not optimal. when we use variables, it is searched only once, during the assignment, and then only used. it is important that you search through DOM in document.ready (which you already do), but it looks nice and readable when we move the variables to the top of that function.